openvm/src/customize_exe.rs

```diff
@@ -145,71 +156,70 @@ pub fn customize(
         g: OpenVmField::<BabyBearField>::ZERO,
     };
 
-    let mut extensions = Vec::new();
-    let n_acc = config.autoprecompiles as usize;
-    let n_skip = config.skip_autoprecompiles as usize;
-
     tracing::info!("Adjust the program with the autoprecompiles");
 
-    // now the blocks have been sorted by cost
-    for acc_block in blocks.iter().skip(n_skip).take(n_acc) {
```

Comment:
No need to skip and take anymore, `blocks_with_apcs` contains everything we need